pom.xml

```diff
@@ -37,6 +37,11 @@
             <artifactId>org.eclipse.paho.client.mqttv3</artifactId>
             <version>1.2.0</version>
         </dependency>
+        <dependency>
+            <groupId>org.bouncycastle</groupId>
+            <artifactId>bctsp-jdk15</artifactId>
+            <version>1.46</version>
```

Comment:
@Alexander-Paeshin is it the correct version?

Reply:
I've picked it from ra pom as well.

Reply:
No, it is not correct version. The library should be bctsp-jdk15on, version 1.64